src/APlayer.js

```diff
@@ -857,6 +867,8 @@ class APlayer {
      * @param {Array} newMusic
      */
     addMusic(newMusic) {
+        let wasSingle = !this.isMultiple();
```

Comment:
Seems that `wasSingle` is not re-assigned below, please use `const` instead.

Reply:
Handled with commit 1a4ea7a4ed4cacfc9e6686495e1e5b72a8e0e1cd.